CHANGELOG.md

```diff
@@ -10,6 +10,51 @@ Format `<github issue/pr number>: <short description>`.
 * [#2922](https://github.com/kroxylicious/kroxylicious/pull/2922): build(deps): bump kafka.version from 4.1.0 to 4.1.1
 * [#1318](https://github.com/kroxylicious/kroxylicious/issues/1318): Add FilterContext#topicNames to enable filters to retrieve names for topic ids
 * [#2821](https://github.com/kroxylicious/kroxylicious/pull/2821): Fix OauthBearerValidationFilter unnecessarily copying the authentication bytes from an incoming request to a failed response 
+* [#2893](https://github.com/kroxylicious/kroxylicious/pull/2893): Add Subject, replace FilterContext#clientSaslAuthenticationSuccess
+* [#2899](https://github.com/kroxylicious/kroxylicious/pull/2898): Add SaslSubjectBuilder API
+* [#2899](https://github.com/kroxylicious/kroxylicious/pull/2899): Add the Authorizer API
+* [#2903](https://github.com/kroxylicious/kroxylicious/pull/2903): Add an ACL Authorizer implementation
+* [#2909](https://github.com/kroxylicious/kroxylicious/pull/2909): Add an Authorizer Filter that can authorize Topic operations
+* [#2913](https://github.com/kroxylicious/kroxylicious/pull/2913): Add pluggable mTLS Subject publishing
+* [#2904](https://github.com/kroxylicious/kroxylicious/pull/2904): SaslInspection Filter publishes Subject using pluggable SaslSubjectBuilder
+
+### Changes, deprecations and removals
+
+* `Subject`, `Principal` and `User` principal added to `io.kroxylicious.proxy.authentication` package.
+* `Subject authenticatedSubject();` added to `FilterContext`, enabling Filters to access the current authenticated Subject.
+* `io.kroxylicious.proxy.authentication.SaslSubjectBuilder` has been added to `kroxylicious-api`. This is an optional
+  Service interface that SASL-oriented Filters can choose to load.
+* `FilterContext#clientSaslAuthenticationSuccess(String mechanism, String authorizedId)` is deprecated. Use
+  `FilterContext#clientSaslAuthenticationSuccess(String mechanism, Subject subject)` instead. Initially the framework
+   expects the Subject to contain a single `io.kroxylicious.proxy.authentication.User` principal which contains the
+   `authorizedId`, though this may change in the future.
+* A Virtual Cluster now has a pluggable `io.kroxylicious.proxy.authentication.TransportSubjectBuilder` associated with it.
+  This new Service is responsible for building a `Subject` from mTLS certificates presented by the client to the proxy.
+  This is configurable on the virtual cluster using the `subjectBuilder`:
+  ```yaml
+  virtualClusters:
+    - name: demo
+      subjectBuilder:
+        type: YourSubjectBuilderType
```

Comment:
I opted to have arbitrary types/config in the examples, I don't want to go deep into documenting the mechanisms, just point out that new fields have landed in the configuration files.

Reply:
I was just mulling that one over. The changelog is not meant to be authoritative docs so some kind of exemplar is probably the right call.